Common/TableProducer/eventSelection.cxx

```diff
@@ -209,6 +215,50 @@ struct BcSelectionTask {
       triggerBcShift = (run <= 526766 || (run >= 526886 && run <= 527237) || (run >= 527259 && run <= 527518) || run == 527523 || run == 527734 || run >= 534091) ? 0 : 294;
     }
 
+    // extract run number and related information
+    int run = bcs.iteratorAt(0).runNumber();
```

Comment:
There is something left: this will SEGV in case the bc table is empty! 

I think one needs to add
```
if (bcs.size() == 0) {
  return;
}
```

Anyone online to do this before this nightly?

Reply:
@ekryshen @altsybee @ddobrigk

Reply:
I'm here, let me take a look right away ...

Reply:
@jgrosseo adding it before l219 would not have captured the first use of `bcs.iteratorAt(0)` so I added it at the very beginning of the process function in #4883

Reply:
Based on usual compilation times and current queue, I think this should make it into tonight's tag. Thanks for catching that @jgrosseo !

Reply:
Thanks a lot for the fact reaction! So that bug was already there before? Maybe we never have empty bcs...

Reply:
Yes, it was already before, it seems ... And better to be safe :-) have a nice evening!